### ISSUE TITLE
Audit shouldn't ignore offscreen focusable elements with aria-hidden

### ISSUE THREAD

**Issue by robdodson** [GoogleChrome/accessibility-developer-tools#273]
Visit this page and resize the browser till it gets into a responsive mobile state
https://www.getmdl.io/templates/dashboard/index.html

The element #accbtn is offscreen but focusable, which should be picked up by the audit but it seems to get a pass because it is also aria-hidden

https://github.com/GoogleChrome/accessibility-developer-tools/blob/582ba71c124ed834d640ce2109dcd91b7864922f/src/js/AccessibilityUtils.js#L585
